samples/tab-deeplink/nodejs/pages/ExtendedDeepLink.html

```diff
@@ -0,0 +1,217 @@
+                            <form>
+                                <span>Add Details for scheduling dialog</span>
+                                <div class="inputContainer">
+                                    <label for="Subject">Subject:</label>
+                                    <input type="text" id="Subject" value="" required />
+                                </div>
+                                <div class="inputContainer">
+                                    <label for="Content">Content:</label>
+                                    <input type="text" id="Content" value="" />
+                                </div>
+                                <div class="inputContainer" style="display:flex;">
+                                    <label for="Attendee">Attendees: </label>
+                                    <input type="text" id="Attendee" value="" disabled required />
+                                    <button onclick="selectPeople()" style="margin:0px 4px;">
+                                        Select
+                                    </button>
+                                </div>
+                                <div class="inputContainer">
+                                    <label for="StartTime">Start Time:</label>
+                                    <input type="datetime-local" id="StartTime" value="" required />
+                                </div>
+                                <div class="inputContainer">
+                                    <label for="EndTime">End Time:</label>
+                                    <input type="datetime-local" id="EndTime" value="" required />
+                                </div>
+                                <div class="inputContainer">
+                                    <span id="dialogDeeplink"></span>
+                                </div>
+                            </form>
+                        </div>
+                    </div>
+                    <button data-id="singleSelect" type="submit" class="callDeeplinkButton"
+                        style="width:130px;margin-left:8.5rem;" onclick="onSchedulingDialogClick()">
+                        Schedule Meeting
+                    </button>
+                </div>
+            </div>
+            <div>
+                <div class="card">
+                    <div class="card-body">
+                        <div class="card-header">
+                            <b>Open a Polly app install dialog</b>
+                        </div>
+                        <div class="configure-header">
+                            To use this option, configure the function using following
+                            parameters
+                            <!-- Since we are using pre tag for having the code snippet in specified format ignoring the indentation here -->
+                            <pre>
+microsoftTeams.app.openLink("https://teams.microsoft.com/l/app/1542629c-01b3-4a6d-8f76-1938b779e48d");
```

Comment:
I think we should not hardcode app id, rather than add place holder and add a note in readme?